io.github.Torrent-file-editor.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.torrent-file-editor
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+
+command: torrent-file-editor
+modules:
+  - name: torrent-file-editor
+    sources:
+      - type: archive
+        url: https://github.com/torrent-file-editor/torrent-file-editor/releases/download/v0.3.17/torrent-file-editor-0.3.17.tar.gz
+        sha256: e81a8701f04715dd9162a67cb1732f9d93209200a2aefcf9d2cc6acbb8d76443
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/torrent-file-editor/torrent-file-editor/releases/latest
+          version-query: '.tag_name | sub("^v"; "")'
+          timestamp-query: .published_at
+          url-query: '.assets[] | select(.name=="torrent-file-editor-" + $version + ".tar.gz") | .browser_download_url'
+      - type: file
+        path: io.github.Torrent-file-editor.metainfo.xml
+      - type: file
+        path: io.github.Torrent-file-editor.desktop
+    buildsystem: "cmake"
+    config-opts:
+      #- -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib
```

Comment:
```suggestion
```

This is not necessary

Reply:
fixed here: https://github.com/flathub/flathub/pull/3457/commits/72b76e51b2b55ae47f7296e3dcbc87a555ab704b